.github/workflows/cwf-operator.yml

```diff
@@ -9,7 +9,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: cwf-operator
+name: CWAG Lint & Test Operator
```

Comment:
I suppose CWF is broader than CWAG because the former refers to the WiFi while the latter refers to the gateway only. I think in the case of the integration test workflow `cwf-integ-test.yaml`, using CWAG as the component makes sense in analogy with AGW for the LTE integ tests. However, perhaps for the operator, CWF does make sense. The operator is referred to as the "CWF Operator" [here](https://github.com/magma/magma/blob/master/cwf/k8s/cwf_operator/README.md), for example. Any thoughts on this? CC @sebathomas @crasu

Reply:
Isn't the operator just running things on the gateway (or basically running the gateway)?

In any case, we should have everything related to carrier wifi under a single top-level component name. Maybe CWF would be better than CWAG but I have no strong preference.

Reply:
For me both is fine it should just be consistent

Reply:
Okay, thanks, then I propose we stick with CWAG